docs/core/compatibility/networking/9.0/enumconverter.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Breaking change: EnumConverter validates registered types to be enum"
+description: Learn about the .NET 9 breaking change in networking where EnumConverter now validates that the type to be registered is an enum type.
+ms.date: 10/04/2024
+---
+# EnumConverter validates registered types to be enum
+
+<xref:System.ComponentModel.EnumConverter> is a type converter that converts to and from an `enum` type. EnumConverter now validates that the type to be registered is of an `enum` type.
+
+## Previous behavior
+
+Previously, the type to be registered was not validated to be an `enum` type.
+
+## New behavior
+
+Starting in .NET 9, <xref:System.ComponentModel.EnumConverter> throws an <xref:System.ArgumentException> if the type to be converted is not an `enum` type. Any derived classes of <xref:System.ComponentModel.EnumConverter> should also respect this requirement.
+
+## Version introduced
+
+.NET 9 Preview 7
+
+## Type of breaking change
+
+This change can affect [binary compatibility](../../categories.md#binary-compatibility) and is also a [behavioral change](../../categories.md#behavioral-change).
```

Comment:
Usually this would just be considered a behavioral change. Is it really binary incompatible?

Reply:
Any assembly that extends an `EnumConverter `(for example, [CommonLocalizableEnumConverter ](https://github.com/microsoft/sqlmanagementobjects/blob/f185f0170573aa1c8ae9fb59eaa4ad602e9de95f/src/Microsoft/SqlServer/Management/ConnectionInfo/LocalizableTypeConverter.cs#L545) will now throw if the [type to be converted](https://github.com/microsoft/sqlmanagementobjects/blob/f185f0170573aa1c8ae9fb59eaa4ad602e9de95f/src/Microsoft/SqlServer/Management/ConnectionInfo/LocalizableTypeConverter.cs#L648-L649) is not an `enum`)

@ericstj

Reply:
This is Behavioral, not binary breaking.  Binary breaking is very rare.  It means binaries built against old assemblies won't work on new.